assets/src/components/map/tooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { Tooltip } from "react-leaflet"
+
+interface MapTooltipProps {
+  title: string
+  permanent?: boolean
+}
+
+export const MapTooltip = ({ title, permanent }: MapTooltipProps) => {
+  return (
+    <Tooltip
+      key={`tooltip-permanent-${permanent}`}
+      className="c-detour_map--tooltip"
```

Comment:
If this is becoming a reusable component, maybe we should change this class to be it's own component, `c-map-tooltip`?

Reply:
https://github.com/mbta/skate/pull/2366/commits/2b518692c199a95ffb667592e3b3d4342943429b